presto-docs/src/main/sphinx/installation/spark.rst

```diff
@@ -0,0 +1,44 @@
+=========================
+Executing Presto on Spark
+=========================
+
+When there is a need to execute long running queries, Presto offers
+an option to scale the resource management using Spark, as it doesn'f offer
+one currently by itself. This way Presto can be executed as a stand alone 
+application on Spark.
+
+Steps
+-----
+
+Download the Presto Spark package tarball, :maven_download:`spark-package` 
+and the Presto Spark laucher, :maven_download:`spark-launcher`. Keep both the
+files at, say, *example* directory.
+
+Presto needs``config.properties``. The following is a minimal 
+``config.properties``:
+
+.. code-block:: none
+
+    query.hash-partition-count=10
+    redistribute-writes=false
+ 
+To execute Presto on Spark, first start your Spark cluster, which we will 
+assume have the URL *spark://spark-master:7077*. Keep your 
+time consuming query in a file called, say, *query.sql*.
+
+.. parsed-literal::
+
+    /spark/bin/spark-submit \
+    --master spark://spark-master:7077 \
+    --executor-cores 4 \
+    --conf spark.task.cpus=4 \
+    --class com.facebook.presto.spark.launcher.PrestoSparkLauncher \
+    presto-spark-launcher-\ |version|\ .jar \
+    --package presto-spark-package-\ |version|\ .tar.gz \
+    --config /presto/etc/config.properties \
+    --catalogs /presto/etc/catalogs \
```

Comment:
Could you please add a paragrapth describing this catalog structure (or point it to the section from the Presto documentation https://prestodb.io/docs/current/installation/deployment.html#catalog-properties